vars/ansible.groovy

```diff
@@ -228,7 +228,9 @@ void installRoles(Map config) {
             commandBuilder.addArgument("--force")
         }
         log.debug("command", commandBuilder.build())
-        sh(commandBuilder.build())
+        retry(3) {
```

Comment:
we should do some logging here to identity the retries.
Something like:
`running ansible-galaxy (1/3)`